cmake/external/featurizers.cmake

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# This source code should not depend on the onnxruntime and may be built independently
+
+set(featurizers_URL "https://github.com/microsoft/FeaturizersLibrary.git")
+set(featurizers_TAG "006df6bff45dac59d378609fe85f6736a901ee93")
+
+set(featurizers_pref FeaturizersLibrary)
+set(featurizers_ROOT ${PROJECT_SOURCE_DIR}/external/${featurizers_pref})
+set(featurizers_BINARY_DIR ${CMAKE_CURRENT_BINARY_DIR}/external/${featurizers_pref})
+
+# Only due to GIT_CONFIG
+# Uncoment UPDATE_COMMAND if you work locally
+# on the featurizers so cmake does not undo your changes.
+if (WIN32)
+    ExternalProject_Add(featurizers_lib
+            PREFIX ${featurizers_pref}
+            GIT_REPOSITORY ${featurizers_URL}
+            GIT_TAG ${ngraph_TAG}
+            # Need this to properly checkout crlf
```

Comment:
NGRAPH?

Reply:
Good catch! May be that's the reason for some troubles.

---
In reply to: [357837997](https://github.com/microsoft/onnxruntime/pull/2643#discussion_r357837997) [](ancestors = 357837997)